src/JuliusSweetland.OptiKey.Core/OptiKeyApp.cs

```diff
@@ -115,6 +115,18 @@ protected void Initialise()
                 Settings.Default.Reload();
             }
 
+            //Migrate the completion time setting to the new field
+            var timespan = Settings.Default.KeySelectionTriggerFixationDefaultCompleteTime;
+            if (timespan> TimeSpan.Zero)
+            {
+                Settings.Default.KeySelectionTriggerFixationDefaultCompleteTimes = timespan.TotalMilliseconds.ToString()
+                    + "," + Math.Round(0.75 * timespan.TotalMilliseconds).ToString()
```

Comment:
I'm aiming to look at this PR (and your other one) in the next few days. Can you clarify what this logic is doing?

Reply:
I can clarify the comment before finalizing the PR, but the intent here was to convert the old single value Completion Time into a list with the new format to nudge users toward exploring the feature. Another idea is to add conditional highlighting to the field in MC and show a message that repeat rates can be added.